### PR TITLE
Add Unofficial Middlewares

### PR DESCRIPTION
**Pull request by fadeenk**
Thought it would be nice to have a list of running middlewares similar to the community plugins on primus. 

Added my rooms middleware, in case someone else would like to use it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,18 @@ function middleware(primus, parse, req, res, next) {
 }
 ```
 
+### Unofficial middlewares
```
Comment (lpinca) [inline README.md]: Plural of middleware is middleware :)

File: README.md [hunk 2]
```diff
@@ -189,6 +189,18 @@ function middleware(primus, parse, req, res, next) {
...
+#### Omega Supreme Rooms Middleware
+
+[![Version npm](https://img.shields.io/npm/v/omega-supreme-rooms-middleware.svg?style=flat-square)](http://browsenpm.org/package/omega-supreme-rooms-middleware)
+[![Build Status](https://img.shields.io/travis/fadeenk/omega-supreme-rooms-middleware/master.svg?style=flat-square)](https://travis-ci.org/fadeenk/omega-supreme-rooms-middleware)
+[![Dependencies](https://img.shields.io/david/fadeenk/omega-supreme-rooms-middleware.svg?style=flat-square)](https://david-dm.org/fadeenk/omega-supreme-rooms-middleware)
+[![Coverage Status](https://img.shields.io/coveralls/fadeenk/omega-supreme-rooms-middleware/master.svg?style=flat-square)](https://coveralls.io/r/fadeenk/omega-supreme-rooms-middleware?branch=master)
+
+This middleware adds support for `primus-rooms` and automatically integrates with `metroplex` if included.
+
+
```
Comment (lpinca) [inline README.md]: Can you please remove one of these blank lines? Keep only one.

File: README.md (hunk 1 above)
Comment (fadeenk) [inline README.md]: had to double check that and found this awesome site http://english.stackexchange.com/questions/257120/middleware-vs-middlewares

**Comment by lpinca** [comment]
Thanks!

**Comment by fadeenk** [comment]
Anytime :) 
If there is anything else that needs to be done let me know

**Pull request merged**